packages/core/src/types.ts

```diff
@@ -3,6 +3,12 @@ import { State } from './State';
 import { Interpreter, Clock } from './interpreter';
 import { IsNever, Model, Prop } from './model.types';
 
+declare global {
+  interface SymbolConstructor {
+    readonly observable: symbol;
+  }
+}
```

Comment:
I don't think a global type should be added by us here. It's not our responsibility to do this on behalf of our users and this is risking some conflicts with other types that may be loaded implicitly and extending the same thing.

Reply:
Weird, I believe we added this in the `Interpreter.ts` module when first looking into interop observables, to make TS believe the existence of `Symbol.observable`, but somehow it now doesn't seem to be necessary anymore (TS version bump?). Should not matter for the user, since the global type is not exported, but I'll try to remove them anyhow.